lib/invoice_printer/document.rb

```diff
@@ -81,7 +81,8 @@ class InvalidInput < StandardError; end
                 :account_swift,
                 # Collection of InvoicePrinter::Invoice::Items
                 :items,
-                :note
+                :note,
+                :background
```

Comment:
We actually don't need it in here. Background will be set only for `PDFDocument` (same as with logo for instance).